src/App.js

```diff
@@ -64,10 +59,16 @@ class App extends PureComponent {
 
 App.propTypes = {
   isAuthenicated: PropTypes.bool,
-}
+};
 
 const mapStateToProps = state => ({
   isAuthenticated: state.isAuthenticated,
-})
+});
+
+const mapDispatchToProps = {
+  setCurrentUser,
+  setNewUsers,
+  fetchList,
+};
 
-export default connect(mapStateToProps, { setCurrentUser, setNewUsers, fetchList })(App);
```

Comment:
I don't think I've ever seen objects proved as `mapDispatchToProps`, changed here

Reply:
pretty sure you taught me this xP

Reply:
if not, then it's possible I grabbed it from a Traversy tutorial.